src/python/genny/csv2.py

```diff
@@ -84,10 +84,6 @@ def __next__(self):
 
     def _parse_into_intermediate_csv(self, line):
         for i in range(len(line)):
-
-            # Strip spaces from lines
-            line[i] = line[i].strip()
-
             # Convert string digits to ints
             if line[i].isdigit():
```

Comment:
FWIW, the `str.isdigit()` approach performs a lot faster than the `try/except ValueError` approach.

```
$ python3 -m timeit '
a = "1"
if a.isdigit():
    int(a)
'
1000000 loops, best of 5: 295 nsec per loop

$ python3 -m timeit '
a = "a"
if a.isdigit():
    int(a)
'
5000000 loops, best of 5: 73.4 nsec per loop

$ python3 -m timeit '
a = "1"
try:
    int(a)
except ValueError:
    pass
'
1000000 loops, best of 5: 241 nsec per loop

$ python3 -m timeit '
a = "a"
try:
    int(a)
except ValueError:
    pass
'
200000 loops, best of 5: 1.33 usec per loop
```